### ISSUE TITLE
Gekko discussion forum?

### ISSUE THREAD

**Issue by jasurakk** [askmike/gekko#1256]
Sorry for a potential offtopic (I understood that this wouldn't be the correct place for newbie questions, so remove this question if it violates the rulez), but is there any good discussion forum for Gekko related questions, problem solving threads, FAQ, etc. anywhere?? Sure, there is this Discord- channel, but I find it to be more or less a "chat-line" where everybody writes whatever comes into their minds, and it is really hard to follow the flow of comments. Or search for a specific topic from the history. I tried to google for a Gekko discussion forum, but did not find any..?

**Comment by askmike** [comment]
There is no forum as of yet, maintaining a forum requires a lot of time (which I currently do not have).

**Comment by jasurakk** [comment]
Yeah, maintaining a moderated forum will surely take some time, that's for sure. But in the other hand; even non-moderated discussion forum would be better than no forum at all. Strong and active community is often the most important thing for any open source project to succeed, because there isn't any commercial support for asking the "stupid" questions just to get started. 

I already tried CryptoTrader and Zenbot for automizing my trading methods, but unfortunately they both seem to have too much problems and restrictions. CryptoTrader totally lacks of documentation and support (and I also hated that "coffee script"), and Zenbot currently has way too much bugs and it seems to be more or less a platform for using the readymade strategies, not so much for creating your own ones. 

I really like the idea of Gekko, where you can easily design & implement your own strategy. So all my respect and gratitude goes for all you guys & gals who have been working year after year for this great project! :=) 

Do you think that there would be any **good** discussion forums regarding crypto bots anywhere in the internet? If it's not possible to have an own forum, maybe Gekko users and contributors could just decide the forum where they meet and share their questions and all other information?

Or, do you think that it would be a totally bad idea to allow questions & other discussions here in Github? Zenbot has it this way, and it seems to be going just fine with them.

**Comment by russelltrafford** [comment]
Slack community anyone? 😃

**Comment by russelltrafford** [comment]
Ah never mind, I just saw https://forum.gekko.wizb.it

**Comment by dewulfbart** [comment]
this could be closed?

**Comment by dewulfbart** [comment]
@jasurakk

**Comment by askmike** [comment]
yea let's do it!
